src/lib/utils/solidFetch.js

```diff
@@ -90,6 +90,30 @@ export const getBasicPod = async webId => {
   }
 };
 
+export const getIdpFromWebId = async webId => {
+  try {
+    let idp = null;
+    if (webId) {
+      const idpConfigUrl = `${new URL(webId).origin}/.well-known/openid-configuration`;
+      // TODO: This could be parallelized
+      const issuer = await data[webId]['solid:oidcIssuer'];
+      console.log(`issuer: ${issuer}`);
+      const idpConfig = await auth.fetch(idpConfigUrl);
```

Comment:
Isn't it better to only perform this request if `issuer` is not truthy?

Reply:
That is absolutely correct. I'll fix this right away.